### ISSUE TITLE
IE 11 Support

### ISSUE THREAD

**Issue by pkelleter** [jiayihu/ng-animate#5]
Hey,
It seems this library does not support IE11, although angular/animations does provide the basis to do so.
The translate property seems to be defined in a non-compatible fashion, since the translateX / translateY functionality does not work properly in IE11.
It would be awesome if that could be fixed, since unfortunately a lot of people still need to support that version.

**Comment by jiayihu** [comment]
Could you give me a hint about the unsupported property? `translate3d` seems to be supported on IE11 according to [caniuse](https://caniuse.com/#search=translate3d)

**Comment by allisonthetaung** [comment]
Hi,
When I run the demo on IE11, all the elements don't come out. 
Does it mean angular animation doesn't support IE at all?
Cheers

**Comment by pkelleter** [comment]
Angular animations needs corresponding polyfills to be activated for ie

allisonthetaung <notifications@github.com> schrieb am Fr., 22. Juni 2018,
01:54:

**Comment by jiayihu** [comment]
From the officiale doc for [Angular animations](https://angular.io/guide/animations#overview):
```
As of Angular 6, If the Web Animations API is not supported natively by the browser, then Angular will use CSS keyframes as a fallback instead (automatically). This means that the polyfill is no longer required unless any code uses AnimationBuilder. If your code does use AnimationBuilder, then uncomment the web-animations-js polyfill from the polyfills.ts file generated by Angular CLI.
```

So if you use Angular 4+ you'll need to include `web-animations-js` polyfill, otherwise it should be fine.

**Comment by amitport** [comment]
note that currently this lib has 'web-animations-js' peer dependency. This probably should be removed as it's not directly a concern of this library and is not required for Angular6+chrome/firefox

(ideally it should have been a peer dependency of `@angular/animations@4`)

**Comment by jiayihu** [comment]
Good point @amitport , I'll fix it asap

**Comment by jiayihu** [comment]
Solved by #13
